packages/standards/biome.jsonc

```diff
@@ -33,17 +33,22 @@
         // be the case. https://biomejs.dev/linter/domains/#project-rules
         "noFloatingPromises": "error",
 
+        "noSecrets": "error",
```

Comment:
Feels like a good thing to have on!

https://biomejs.dev/linter/rules/no-secrets/

Reply:
lmao